newIDE/app/src/ProjectsStorage/LocalProjectWriter.js

```diff
@@ -22,7 +22,7 @@ export default class LocalProjectWriter {
         return;
       }
 
-      LocalProjectWriter._writeProjectJSONFile(project, filepath, err => {
+      LocalProjectWriter.writeProjectJSONFile(project, filepath, err => {
```

Comment:
After thinking about it, I think we can:

* have a `autosaveProject` in LocalProjectWriter (that you pass to the MainFrame with onAutoSaveProject={LocalProjectWriter.autosaveProject).
* have a `checkAutosave` in LocalProjectOpener (which verify if an autosave can/should be loaded, and that you pass to the MainFrame with some "onCheckAutosave" prop :))

Reply:
How about calling it `shouldOpenAutosave` ? The check compares it it exists, but also the dates. When the file is corrupted, you don't want it to check the dates. Seems like a clearer name, since it should also return a boolean